io.github.kraflab.DSDA-Doom.yml

```diff
@@ -0,0 +1,55 @@
+id: io.github.kraflab.DSDA-Doom
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: dsda-doom
+modules:
+- name: libzip
+  buildsystem: cmake-ninja
+  config-opts:
+  - -DCMAKE_BUILD_TYPE=Release
+  sources:
+  - type: git
+    url: https://github.com/nih-at/libzip
+    tag: v1.11.3
+- name: glu
+  buildsystem: meson
+  config-opts:
+  - --buildtype=release
+  sources:
+  - type: git
+    url: https://gitlab.freedesktop.org/mesa/glu
+    tag: glu-9.0.3
+- name: fluidsynth
```

Comment:
pleqase use fluidsynth from the shared-modules https://github.com/flathub/shared-modules/blob/master/linux-audio/fluidsynth2.json